Project.toml

```diff
@@ -2,7 +2,7 @@ name = "PowerModels"
 uuid = "c36e90e8-916a-50a6-bd94-075b64ef4655"
 authors = ["Carleton Coffrin"]
 repo = "https://github.com/lanl-ansi/PowerModels.jl"
-version = "0.17.3"
+version = "0.18.0"
```

Comment:
I have found that it is best to not update the version number until just before release, otherwise I run into compat issues during release resting phase.  So I usually leave the version number as-is until just before merging the breaking features into master.

Reply:
Interesting, I was not aware of this issue. Good to know, and I'll keep this in mind for future pull requests.